src/test/integration/java/com/commercetools/service/ctp/impl/OrderServiceImplIT.java

```diff
@@ -73,6 +74,7 @@ public void getWithWrongPaymentId() {
     }
 
     private void createOrderWithPayment(Payment ctPayment) {
+        ensureTestTaxCategory(sphereClient);
```

Comment:
Optional optimisation: To avoid loading tax category twice you could change return type of [ensureTestTaxCategory](https://github.com/commercetools/commercetools-paypal-plus-integration/blob/master/src/test/integration/java/com/commercetools/testUtil/ctpUtil/TaxUtil.java#L19) from void to TaxCategory and map it from `fetch` or `create` requests.

Reply:
changed the method accordingly db3a1332c1bc5337b6d5d1d4b2a04c8c4f5cecbc

Reply:
I commented on the change. Please check

Reply:
https://github.com/commercetools/commercetools-paypal-plus-integration/pull/142/commits/585ba5f230e54f244cb2b7643e9b6fa32bc7c3e8